hack/vendor.sh

```diff
@@ -38,6 +38,11 @@ clone git github.com/hashicorp/consul v0.5.2
 clone git github.com/docker/distribution 419bbc2da637d9b2a812be78ef8436df7caac70d
 clone git github.com/vbatts/tar-split v0.9.4
 
+clone git github.com/docker/notary 9926ead788e6a5657de62560a4fa25c0a771a2d8
+clone git github.com/endophage/gotuf 374908abc8af7e953a2813c5c2b3944ab625ca68
+clone git github.com/tent/canonical-json-go 96e4ba3a7613a1216cbd1badca4efe382adea337
```

Comment:
omg yes, we can do that! i mean i dont want to but if necessary i will

On Thu, Jul 23, 2015 at 2:24 PM, Arnaud Porterie <notifications@github.com>
wrote:

> In hack/vendor.sh
> <https://github.com/docker/docker/pull/14546#discussion_r35373168>:
>
> > @@ -38,6 +38,11 @@ clone git github.com/hashicorp/consul v0.5.2
> >  clone git github.com/docker/distribution 419bbc2da637d9b2a812be78ef8436df7caac70d
> >  clone git github.com/vbatts/tar-split v0.9.4
> >
> > +clone git github.com/docker/notary 9926ead788e6a5657de62560a4fa25c0a771a2d8
> > +clone git github.com/endophage/gotuf 374908abc8af7e953a2813c5c2b3944ab625ca68
> > +clone git github.com/tent/canonical-json-go 96e4ba3a7613a1216cbd1badca4efe382adea337
>
> Example of what I think would be a better quality dep: a repo holding a
> generator script which git clones the golang standard package with a fixed
> hash/tag and applies a diff. I believe it would make the dependency much
> more auditable: right now it's kind of a black box.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/14546/files#r35373168>.
>